db/backend_test.go

```diff
@@ -2,42 +2,76 @@ package db
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
-func testBackend(t *testing.T, backend string) {
+func cleanupDBDir(dir, name string) {
+	os.RemoveAll(filepath.Join(dir, name) + ".db")
+}
+
+func testBackendGetSetDelete(t *testing.T, backend string) {
 	// Default
 	dir, dirname := cmn.Tempdir(fmt.Sprintf("test_backend_%s_", backend))
 	defer dir.Close()
 	db := NewDB("testdb", backend, dirname)
-	require.Nil(t, db.Get([]byte("")))
```

Comment:
I believe []byte("") and nil are pretty essential keys for unit tests, as they are the blank keys and defaults usually so can help weed out any stale internals.

Reply:
yeah makes sense to add this back